CONTRIBUTING.md

```diff
@@ -59,15 +59,15 @@ Other commit information:
 
 * [How to format the message][commit-message-message]
 
-## How to build the docker image (tag 18.04)
+## How to build the docker image (tag 22.04)
 
 ```bash
-docker build --no-cache --build-arg version=18.04 --build-arg strip=true  -t eclipse/tracecompass-build-env:18.04 .
+docker build --network host --no-cache --build-arg version=22.04 --build-arg strip=true  -t eclipse/tracecompass-build-env:22.04 .
```

Comment:
> --network host 

Did you have a specific reason to add this? This seems to be a new option for "docker build". Di you need it when building locally?

Reply:
I need that for some reason on my laptop.

Reply:
I think it's ok - it should not affect the resulting image, so we can do whatever is needed to successfully build.